test-plan.md

```diff
@@ -92,6 +92,19 @@ The easist way to verify that a project was successfully loaded is to open a .cs
 * In a project that uses globbing (.NET Core), use the VS Code file explorer to add a new file next to the csproj. Intellisense/sighelp/etc should be available in the new file
 * Add a new file and reference a type in it from a different file. Deleting from disk the file containing the referenced type  should produce error messages
 
+#### Omnisharp Options
+
+  #### omnisharp.useMono (for Linux)
```

Comment:
#### omnisharp.useMono (forLinux)
Determines which Mono installation is used to run Omnisharp. Defaults to false.
If true, the host's globally installed Mono is used. Mono must be version 5.2.0 or greater.
If false, the Mono bundled with Omnisharp is used.

Reply:
@TheRealPiotrP As per this description, how will the people who test determine which version of mono is being used- the bundled one or the globally installed one. That is why I mentioned about using the "run" and the "Omnisharp.exe". Thoughts ?

Reply:
sorry, will fix up now :)